node/libs/roles/src/proto/validator.proto

```diff
@@ -3,19 +3,21 @@ syntax = "proto3";
 package zksync.roles.validator;
 
 import "zksync/std.proto";
+import "zksync/roles/attester.proto";
 
 message Genesis {
   reserved 1,2;
   reserved "fork","validators";
   optional uint64 chain_id = 5; // required
   optional uint64 fork_number = 6; // required; ForkNumber
   optional uint64 first_block = 7; // required; BlockNumber
+  repeated attester.WeightedAttester attesters = 4;
  
   // These properties are expected to be overwritten each epoch.
   // We will either remove them entirely, or keep them for the initial epoch.
   optional uint32 protocol_version = 8; // required; ProtocolVersion
   repeated WeightedValidator validators_v1 = 3;
-  optional LeaderSelectionMode leader_selection = 4; // required
+  optional LeaderSelectionMode leader_selection = 9; // required
```

Comment:
the whole point of numeric tags is that they are not allowed to change to maintain backward/forward compatibility. Just use the next unused number for the new field.